src/systemctl/systemctl.c

```diff
@@ -8472,6 +8479,13 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
                         arg_reboot_argument = optarg;
                         break;
 
+                case ARG_TIMESTAMP_STYLE:
+                        if (STR_IN_SET(optarg, "us", "µs"))
+                                arg_timestamp_style = TIMESTAMP_US;
+                        if (streq(optarg, "utc"))
```

Comment:
the parser for this should probably be in some timestamp_style_from_string() function like we usually do it. Given the special `us` vs. `µs` tweak it probably can't be implemented using our usual string-table.h macro magic...

Reply:
yeah the usual pattern doesn't quite work - is it worth the additional complication for just 2 enum values?